src/generics/middleware/authenticator.js

```diff
@@ -472,7 +477,8 @@ module.exports = async function (req, res, next, token = '') {
 
 				let validateOrgsResult = await validateIfOrgsBelongsToTenant(
 					req.headers['tenantid'],
-					req.headers['orgid']
+					req.headers['orgid'],
```

Comment:
@borkarsaish65 can we use camelCase orgId also tenantid it should be in camelcase,